cfgov/jinja2/v1/_includes/molecules/featured-content.html

```diff
@@ -29,7 +29,6 @@
 
 <section class="block
```

Comment:
All the block classes can be removed from this `section` elem, just keep the `featured-content-module`